lib/ansible/modules/network/cloudengine/ce_command.py

```diff
@@ -52,7 +52,6 @@
         by the configured retries, the task fails.  See examples.
     required: false
     default: null
-    aliases: ['waitfor']
```

Comment:
`waitfor` was the original option name in the previous `*_command` network modules. In Ansible 2.2 we standardised on `wait_for`. The alias was added in to allow backwards compatibility,

As this is a new module you only need to support the new option of `wait_for`